EspressoSystems/espresso-sequencer
# refactor!: PlonkVerifier public input reduced from 8 to 4

### This PR:

As described by the title

### How to test this PR: 

Since there are many other pending changes that fail CI, to test the correctness of this change:
```
cargo build --release --bin diff-test
forge test --mc PlonkVerifier
```

all plonk related tests should pass
